eva_submission/config_migration.py

```diff
@@ -9,6 +10,27 @@
 logger = log_cfg.get_logger(__name__)
 
 
+def upgrade_version_1_15_to_1_16(eload_cfg, eload_dir):
+    """
+    Upgrades a version 1.15 directory structure to version 1.16 to change the path to ingestion nextflow directories
+    """
+    project_dir = eload_cfg.query('ingestion', 'project_dir')
+    if not project_dir:
+        # This ELOAD never went through ingestion so there are no need to change anything
```

Comment:
Setting the version to 1.16 is the responsibility of the ELOAD object. Since the migration is called from the Eload. upgrade_to_new_version_if_needed, it will be updated when the Eload object is closed.

Reply:
Okay, I was just going off the other `upgrade_version` methods (e.g. lines 46 and 86 below). If it's not needed there either it should probably be removed.

Reply:
no need for this PR though